### PR TITLE
Cis 1.2 changes

### LINKED ISSUE #13: CIS v1.2 Req 1: IAM
Please add policy coverage for the following sections:

- 1.16: Ensure IAM policies are attached only to groups or roles
- 1.19: Ensure IAM instance roles are used for AWS resource access from instances
- 1.20: Ensure a support role has been created to manage incidents with AWS Support
- 1.21: Do not setup access keys during initial user setup for all IAM users that have a console password
- 1.22: Ensure IAM policies that allow full "*:*" administrative privileges are not created

### PR DESCRIPTION
**Pull request by ak-dassana**
Added a series of changes to the policies of the vendors PrismaCloud and SecurityHub. Closes #13.

### CODE REVIEW

File: content/workflows/policies/csp/aws/service/iam/resources/policy/iam-support-policy-exists.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+schema: 1.0
+labels:
+  - contextualization
+type: contextualize
+category: iam
+subcategory: authorization
+id: "iam-support-policy-exists"
+name: "IAM Support Policy Exists"
```
Comment (parth-dassana) [inline content/workflows/policies/csp/aws/service/iam/resources/policy/iam-support-policy-exists.yaml:8]:
```suggestion
id: "iam-support-role-exists"
name: "IAM Support Role Exists"
```

File: content/workflows/policies/csp/aws/service/iam/resources/user/iam-user-policy-exists.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+schema: 1.0
+labels:
+  - contextualization
+type: contextualize
+category: iam
+subcategory: authentication
```
Comment (parth-dassana) [inline content/workflows/policies/csp/aws/service/iam/resources/user/iam-user-policy-exists.yaml:6]: @dassana-junix / @ak-dassana - I feel this is more of an `authorization` issue. Do you guys disagree?

**Review by parth-dassana — CHANGES_REQUESTED** [review]
Looks great! Just a few comments.

File: content/workflows/policies/csp/aws/service/ec2/resources/instance/iam-role-not-enabled.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+schema: 1.0
+labels:
+  - contextualization
+type: contextualize
+category: networking
+subcategory: firewall
```
[the remark this answers is not in the document]
Comment (dassana-junix) [reply content/workflows/policies/csp/aws/service/ec2/resources/instance/iam-role-not-enabled.yaml:6]: IAM.Authorization seems to be the more valid categorization here.

File: content/workflows/policies/csp/aws/service/iam/resources/user/iam-user-policy-exists.yaml (hunk 2 above)
Comment (dassana-junix) [reply content/workflows/policies/csp/aws/service/iam/resources/user/iam-user-policy-exists.yaml:6]: It's authorization necause IAM user policy is about what permissions user has, authentication would be more of an access key related policy.

File: content/workflows/policies/csp/aws/service/iam/resources/policy/iam-no-admin-access.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+schema: 1.0
```
Comment (parth-dassana) [inline content/workflows/policies/csp/aws/service/iam/resources/policy/iam-no-admin-access.yaml:1]: Please rename this file to match the id of the policy

File: content/workflows/policies/csp/aws/service/iam/resources/policy/iam-role-not-enabled.yaml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+schema: 1.0
+labels:
+  - contextualization
+type: contextualize
+category: iam
+subcategory: authorization
+id: "iam-role-not-enabled"
```
Comment (parth-dassana) [inline content/workflows/policies/csp/aws/service/iam/resources/policy/iam-role-not-enabled.yaml:7]:
```suggestion
id: "iam-role-not-enabled-for-ec2"
```

Please also rename the file

File: content/workflows/policies/csp/aws/service/iam/resources/policy/iam-support-policy-exists.yaml [hunk 6]
```diff
@@ -0,0 +1,26 @@
+schema: 1.0
```
Comment (parth-dassana) [inline content/workflows/policies/csp/aws/service/iam/resources/policy/iam-support-policy-exists.yaml:1]: Please rename file to match the id

**Review by parth-dassana — CHANGES_REQUESTED** [review]
Couple more!

**Review by parth-dassana — APPROVED** [review]
LGTM! Nice work!

**Pull request merged**